### PR TITLE
font styling issue resolved

### PR DESCRIPTION
**Pull request by HimanshuNarware**
##Description
Resolve font styling issues also added hover effect on icons and heading

issue code
#282
##video

https://github.com/JasonDsouza212/free-hit/assets/83147410/eca91fbe-213d-466d-90bb-fdd04dc19a94

### CODE REVIEW

File: src/styles/Home.css [hunk 1]
```diff
@@ -48,10 +48,12 @@ body {
   grid-template-columns: repeat(auto-fit, minmax(320px, auto));
   align-items: start;
   justify-content: center;
-  gap: 1rem 1rem;
+  /* gap: 1rem 1rem; */
```
Comment (VenomFate-619) [inline src/styles/Home.css:51]: remove comments

File: src/styles/Home.css [hunk 2]
```diff
@@ -83,14 +85,54 @@ img {
...
+  width:0;
+  height: 3px;
+  background-color: var(--primary-black);
+  bottom: -5px;
+  left: 50%;
+  transition: .5s;
+  transform-origin: center;
+
 
+  /* border:2px solid black;
```
Comment (VenomFate-619) [inline src/styles/Home.css:112]: remove comments

Comment (HimanshuNarware) [reply src/styles/Home.css]: comments removed

**Comment by JasonDsouza212** [comment]
Branch has conflicts @HimanshuNarware

**Comment by HimanshuNarware** [comment]
> Branch has conflicts @HimanshuNarware

All clear now

**Comment by JasonDsouza212** [comment]
The design is unsatisfying because the gap between the title and the image is too large. Additionally, there is a hover effect on the image that serves no purpose. One solution is to add a hover effect to both the image and the title simultaneously, while also removing the initial shadow effect.

**Comment by HimanshuNarware** [comment]
> The design is unsatisfying because the gap between the title and the image is too large. Additionally, there is a hover effect on the image that serves no purpose. One solution is to add a hover effect to both the image and the title simultaneously, while also removing the initial shadow effect.

ok

**Comment by HimanshuNarware** [comment]
@JasonDsouza212 hey what you think about this

https://github.com/JasonDsouza212/free-hit/assets/83147410/06d0c9ef-b868-4a2c-b63a-fe9576515aeb

**Comment by JasonDsouza212** [comment]
Looks good, @HimanshuNarware, but can you reduce the image scaling when hovered?

**Comment by HimanshuNarware** [comment]
> Looks good, @HimanshuNarware, but can you reduce the image scaling when hovered?

ok

**Comment by JasonDsouza212** [comment]
@HimanshuNarware,sorry for not replying

**Comment by JasonDsouza212** [comment]
Also attach a video with it

**Comment by HimanshuNarware** [comment]
> Also attach a video with it


https://github.com/JasonDsouza212/free-hit/assets/83147410/b01a0c9a-8588-45cf-9493-cafd377674a3

**Comment by JasonDsouza212** [comment]
Let's keep the card styling the same as it is in our Free-hit. The only change required is the hover effect. When hovering over the title, the effect should be applied to both the image and the title. Additionally, there is an error due to the missing closing curly bracket '}', and it is causing the height of the card to be incorrect.
Screenshot from 2023-06-21 18-49-22

**Pull request closed without merging**